blocks/library/pullquote/index.js

```diff
@@ -90,6 +90,7 @@ export const settings = {
 							value: fromRichTextValue( nextValue ),
 						} )
 					}
+					/* translators: the text of the quotation */
```

Comment:
I don't think this is the way to go to pass context for the translators. I bet we need to extend [__()](https://github.com/WordPress/gutenberg/blob/master/i18n/index.js#L43) to support context param.  @aduth can you confirm?